src/components/Dialogs/Conditions/ConditionsManager.js

```diff
@@ -73,15 +75,15 @@ const ConditionsManager = ({
     onClose,
     setConditionsByDimension,
 }) => {
-    const dimensionType = dimension.valueType
+    const valueType = dimension.valueType
     const isOptionSetCondition =
-        dimensionType === DIMENSION_TYPE_TEXT && dimension.optionSet
+        valueType === DIMENSION_TYPE_TEXT && dimension.optionSet
 
     const [conditionsList, setConditionsList] = useState(
```

Comment:
This piece makes me a bit dizzy..

Reply:
Haha how so? Suggestions welcome 🙂  the logic is that option set conditions are `type === TEXT` + has an `optionSet`

Reply:
I mean the entire block of logic